src/amo/components/Header/index.js

```diff
@@ -149,6 +149,7 @@ export class HeaderBase extends React.Component {
     ) : (
       <AuthenticateButton
         className="Header-authenticate-button Header-button"
```

Comment:
Its no more a button so I guess we have to change the className to something like `.Header-authentication-link`, rest @bobsilverberg can confirm :)

Reply:
Well, the component is still called `AuthenticateButton`, so I think this className is still ok.